ci_framework/roles/operator_build/tasks/zuul_info.yml

```diff
@@ -34,7 +37,4 @@
           - name: "{{ item.project.short_name }}"
             src: "{{ ansible_user_dir }}/{{ item.project.src_dir }}"
             pr_sha: "{{ item.patchset }}"
-            pr_owner: "{{ result.json.head.repo.full_name }}"
-  when:
-    - "'change_url' in item"
-    - "'operator' in item.project.short_name"
+            pr_owner: "{{ result.json.head.user.login }}"
```

Comment:
this is not the same info as in `repo.full_name`
in repo.full_name we get `{github_user}/{source_repo_name}`
and with login you are only getting the` {github_user}`

It was originally getting from `'repo.full_name'` because it is not mandatory for a fork to have the same repo name (we had an issue like that befote, after a operator repo get renamed).

It is failing on molecule job because the PR used in the test had the source repo deleted by its owner (dprince delete his mariadb-operator fork in some moment), see https://github.com/openstack-k8s-operators/mariadb-operator/pull/100

We have 2 options:
 1. update the PR used as example in the molecule job (it is always expected in a zuul job that the source repo exists on an open PR)
 2. Join {gtihub_login} with {short_name}, and only supports PR from forks that have the same name as the original repos.